benchmarks/dynamo/common.py

```diff
@@ -2970,6 +2998,17 @@ def run(runner, args, original_dir=None):
         )
         experiment = speedup_experiment
         output_filename = "inductor.csv"
+    elif args.export:
+        optimize_ctx = functools.partial(
+            torch._dynamo.export,
```

Comment:
The correct API to use should be torch._export.export(). If you check the implementation it uses _dynamo.export internally as well and have extra logic to make sure we preserve all metadata correctly.